openqa-investigate

```diff
@@ -374,7 +374,17 @@ investigate() {
     comment_id=$(sync_via_investigation_comment "$id" "$first_cluster_job_id") || rc=$?
     [[ $rc != 255 ]] && return $rc
 
-    out=$(trigger_jobs "$id" "${@:2}") || return $?
+    echo "------- comment_id: $comment_id"
+    out=""
+    if ! out=$(trigger_jobs "$id" "${@:2}"); then
+        echo "------- out: $out"
+        client-delete-job-comment "$first_cluster_job_id" "$comment_id"
+        echoerr "Unable to trigger investigation jobs for job $id"
```

Comment:
What should the administrator receiving this message do about the error? I don't think this is actionable

Reply:
You are completely right, maybe something like `Unable to complete the investigation on job $id due to clone failure, check for missing repositories or invalid configurations`?

Reply:
Again, what should the *administrator* receiving this message do about the error? You are part of the admin team for o3 and osd so think what *you* would be able to deduce from this message if anything. I think this message is directed more towards test reviewers but they won't see it here. That's why we have AC3

Reply:
Not every log line needs to be actionable by the administrator. The one might still be useful when debugging the script itself. 
Of course it would also make sense to write this message as openQA comment. 
I find the dashes at the beginning of the messages a bit excessive.